Header header
uint8 motor
uint8 job_id
